package/setup.py

```diff
@@ -464,23 +534,28 @@ def dynamic_author_list():
           packages=find_packages(),
           package_dir={'MDAnalysis': 'MDAnalysis'},
           ext_package='MDAnalysis',
-          ext_modules=exts,
+          ext_modules=extensions(config, use_cython),
           classifiers=CLASSIFIERS,
-          cmdclass=cmdclass,
-          requires=['numpy (>=1.5.0)', 'biopython',
-                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)'],
+          distclass=MDADistribution,
+          mda_use_cython=use_cython,
+          requires=['numpy (>=1.5.0)',
+                    'biopython',
+                    'networkx (>=1.0)',
+                    'GridDataFormats (>=0.3.2)',
+                    'six (>=1.4.0)',
+                    ] + require_cython[:1],
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
-          setup_requires=[
+          mda_build_requires=[
               'numpy>=1.5.0',
-          ],
+              ] + require_cython[1:],
```

Comment:
Ah, misunderstood your doubt. As you can see, `requires` and `install_requires` use different version-requirement formats. We have to cater to that. I made our own `mda_build_requires` follow the `install_requires` format.
(And admittedly, put zero effort in checking whether we can use a common format for both, since things were already like this before I touched the code.)